entrypoint_alpine.sh

```diff
@@ -10,6 +10,14 @@ fi
 set -euo pipefail
 IFS=$'\n\t'
 
+# Install Java unlimited strength cryptography
+if [ "${CRYPTO_POLICY}" = "unlimited" ] && [ ! -d "${JAVA_HOME}/jre/lib/security/policy/unlimited" ]; then
+echo "Installing OpenJDK unlimited strength cryptography policy..."
+  apk update
```

Comment:
This update is not needed if you use `--no-cache`, see https://github.com/gliderlabs/docker-alpine/blob/master/docs/usage.md#disabling-cache

Reply:
But I'm not 100% sure whether that is only for add or also for fix

Reply:
The update is required for downloading the package index which is removed during the build. Without it you can not operate on packages.

Reply:
Citing from above link:
> This avoids the need to use --update and remove /var/cache/apk/* when done installing packages.

And I just tested it:

```shell
/ # apk fix --no-cache git
fetch http://dl-cdn.alpinelinux.org/alpine/v3.7/main/x86_64/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/v3.7/community/x86_64/APKINDEX.tar.gz
(1/1) Reinstalling git (2.15.0-r1)
Executing busybox-1.27.2-r6.trigger
OK: 22 MiB in 19 packages
/ # ls -la /var/cache/apk/
total 8
drwxr-xr-x    2 root     root          4096 Dec  1 16:32 .
drwxr-xr-x    6 root     root          4096 Jan 12 08:51 ..
/ #
```